app/javascript/components/widgets/weather/semi-circle.jsx

```diff
@@ -11,7 +11,7 @@ import { colors } from '../../../lib/theme';
 export class SemiCircle extends React.Component {
   static propTypes = {
     totalTime: PropTypes.number.isRequired,
-    sunset: PropTypes.instanceOf(Date).isRequired,
+    endTime: PropTypes.instanceOf(Date).isRequired,
```

Comment:
Curious about the name change -- what were your thoughts about it?

Reply:
I wasn't a hundred percent happy changing it to start and end (thought it was too vague), but sunrise and sunset weren't accurate/confusing and I honestly couldn't think of what would be a better name at the time. Suggestions are welcome!